app/react/hierarchy/keys/Key.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import React from 'react';
+import LabelLink from './LabelLink';
+
+type PropTypes = {
+  dispatch: Function,
+  label: string,
+  // eslint-disable-next-line
+  location: { query: Object },
+  childrenKeys: Array<Object>,
+  style: Object,
+  createStyles: (level: number) => Object,
+  path: Array<string>,
+  setPath: Function
+};
+
+export default function Key({ dispatch, label, childrenKeys, location, style, createStyles, path, setPath }: PropTypes) {
```

Comment:
There are too many arguments. Destructuring outside function declaration will be better, probably.